install.sh

```diff
@@ -121,9 +123,27 @@ elif lspci | grep -E -i '(vga|3d|display)' | grep -i amd > /dev/null; then
 			vulkan-radeon \
 			lib32-vulkan-radeon \
 			xf86-video-amdgpu
+	
+	elif lspci -nnk | grep -i vga -A3 | grep 'Kernel modules: amdgpu' > /dev/null; then
+		echo "AMD graphics card with amdgpu kernel module detected, installing drivers..."
+		pacman --noconfirm -S \
```

Comment:
would be nice to combine this with the list above so we don't have to repeat ourselves. unless we expect this list to diverge in the future?

Reply:
First one is Sea Island and Southern Island, we are not repeating cause modern AMD Graphics cards use only amdgpu kernel module and don't work with radeon module. There are big changes in way that they are built in Linux kernel.

Reply:
this condition and thecone above contain installation of the exact same packages. you could move the package installation up into a condition that catches both gpu cases and then do the configuration only in the one case. that way we dont have to repeat the  list of packages to be installed.

Reply:
It cannot be matched since modern AMD graphics cards don't use radeon kernel module, it would result in error and skipping driver installation.